docs/api.md

```diff
@@ -1162,6 +1162,8 @@ Shortcut for [page.mainFrame().executionContext().queryObjects(prototypeHandle)]
   - `omitBackground` <[boolean]> Hides default white background and allows capturing screenshots with transparency. Defaults to `false`.
 - returns: <[Promise]<[Buffer]>> Promise which resolves to buffer with captured screenshot
 
+> **NOTE** Screenshots take at least 1/6 second on OS X. See https://crbug.com/741689 for discussion.
```

Comment:
It’s also windows

Reply:
@JoelEinbinder I'm unaware about windows; the bug I refer to explains OS X troubles only